helio_tools/_src/utils/time.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+
+def check_datetime_format(datetime_str: str) -> datetime:
+    try:
+        datetime_obj = datetime.strptime(datetime_str, "%Y-%m-%d")
+        return datetime_obj
+    except ValueError:
+        msg = "Datetime string format is incorrect."
+        msg += f"\nInput: {datetime_str}"
+        print(msg)
```

Comment:
Any reason to just print the message? We could do something like 


def check_datetime_format(datetime_str: str) -> datetime:
    try:
        datetime_obj = datetime.strptime(datetime_str, "%Y-%m-%d")
        return datetime_obj
    except ValueError:
        msg = "Datetime string format is incorrect."
        msg += f"\nInput: {datetime_str}"
        raise ValueError(msg)

Reply:
Done!